### ISSUE TITLE
Clicking any notification sends u half way up the chat

### ISSUE THREAD

**Issue by viasux** [ajbura/cinny#479]
### Describe the bug

Clicking any notification sends u half way up the chat

### Reproduction

1. Click notification
2. Observe being scrolled half way up the chat

### Expected behavior

Expected to be at the bottom of the chat

### Platform and versions

```shell
1. Windows 10
2. Librewolf 98.0.2-1
3. 1.8.2 (im.kitties.gay)
4. the-apothecary.club
```

**Comment by viasux** [comment]
actually no this happens even if i recieve a message and then ctrl+k to it

**Comment by kfiven** [comment]
Hi, I am not able to reproduce it on Windows 10 + Librewolf.

**Comment by viasux** [comment]
> Hi, I am not able to reproduce it on Windows 10 + Librewolf.

might be fixed on v2.0

**Comment by ajbura** [comment]
closing as can’t reproduced by myself either.
